diff --git a/scripts/pnglibconf.dfa b/scripts/pnglibconf.dfa
index addbb14..123ddc8 100644
--- a/scripts/pnglibconf.dfa
+++ b/scripts/pnglibconf.dfa
@@ -413,7 +413,7 @@ setting USER_HEIGHT_MAX default       1000000
 
 # Use 0 for unlimited
 setting USER_CHUNK_CACHE_MAX default     1000
-setting USER_CHUNK_MALLOC_MAX default 8000000
+setting USER_CHUNK_MALLOC_MAX default 16000000
 
 # If this option is enabled APIs to set the above limits at run time are added;
 # without this the hardwired (compile time) limits will be used.
